### ISSUE TITLE
Could not load 'guard/yard' or '~/.guard/templates/yard' or find class Guard::Yard

### ISSUE THREAD

**Issue by eprislac** [panthomakos/guard-yard#19]
Happens on trying to run guard init yard

**Comment by panthomakos** [comment]
Can you provide more details - version of Ruby you are running, a Gemfile, etc. Can you provide a sample application or gist that is reproducible?

**Comment by eprislac** [comment]
jRuby 1.7.19 in 1.9 mode, Rails 3. We have guard up and running with Guard-Rspec and Guard-Rubocop. Yard runs fine outside of Guard.
I can’t even get guard-yard to init, because it can’t find the templates.
Cannot provide a gist, it’s an enterprise codebase, not allowed to publicly post any of it, save for what’s in the public repos (not what I’m working with now, which is the main app.

Best Regards,
Eddie

Eddie Prislac
Sr. Software Engineer, NBC Team
t:  (518) 691-0071 x338
prislac@informz.com<mailto:prislac@informz.com>


[cid:image001.png@01D03C95.FF3D3760]


Informz, Inc.
www.informz.com<http://www.informz.com> | Facebook<http://www.facebook.com/informz> | Twitter<http://www.twitter.com/informz> | Blog<http://www.informz.com/blog>

[cid:image002.png@01D03C95.FF3D3760]

On Apr 17, 2015, at 4:54 PM, Pan Thomakos <notifications@github.com<mailto:notifications@github.com>> wrote:


Can you provide more details - version of Ruby you are running, a Gemfile, etc. Can you provide a sample application or gist that is reproducible?

**Comment by panthomakos** [comment]
I tested with `jruby-1.7.19` and `guard init yard` and was not able to reproduce. You can try yourself by cloning this repo and testing it out. I'll need more details or something reproducible to be able to help you resolve the issue. Are you sure you have configured everything correctly? It looks like maybe something is missing from your loadpath. Maybe `bundle exec guard init yard` would help?

**Comment by eprislac** [comment]
no, that's how I'm calling it. I have rvm settings in .ruby-gemset and .ruby-version stipulating where the gems are installed

**Comment by panthomakos** [comment]
Is `guard-yard` part of your `Gemfile`?

**Comment by eprislac** [comment]
Yup.

**Comment by panthomakos** [comment]
Are you able to reproduce the issue if you don't have custom `.ruby-gemset` and `.ruby-version` configurations? Also, can you reproduce the issue on the actual `guard-yard` repo, for example:

```
$ git clone https://github.com/panthomakos/guard-yard.git
$ cd guard-yard
$ bundle
$ bundle exec guard init yard
```

**Comment by eprislac** [comment]
Yeah, init guard on guard-yard works fine. It’s getting it to work in my own projects I have issues with.

Eddie Prislac
Sr. Software Engineer, NBC Team
t:  (518) 691-0071 x338
prislac@informz.com<mailto:prislac@informz.com>


[cid:image001.png@01D03C95.FF3D3760]




Informz, Inc.
www.informz.com<http://www.informz.com> | Facebook<http://www.facebook.com/informz> | Twitter<http://www.twitter.com/informz> | Blog<http://www.informz.com/blog>

[cid:image002.png@01D03C95.FF3D3760]

On May 20, 2015, at 4:18 PM, Pan Thomakos <notifications@github.com<mailto:notifications@github.com>> wrote:


git clone https://github.com/panthomakos/guard-yard.git

**Comment by panthomakos** [comment]
Is there any more information you can provide? I've created the simplest project possible like this, and I cannot reproduce this issue:

```
$ mkdir tmp
$ cd tmp
$ rbenv local jruby-1.7.19
$ echo "source 'https://rubygems.org'\ngem 'guard-yard'" > Gemfile
$ bundle
$ bundle exec guard init yard
``

I get a `Guardfile` that looks like this:

```
guard 'yard' do
  watch(/app\/.+\.rb/)
  watch(/lib\/.+\.rb/)
  watch(/ext\/.+\.c/)
end
```

If you can provide a gist that reproduces this issue, I would be happy to help. You can always try debugging guard itself to see if you can find out what might be causing the issue.
